package.json

```diff
@@ -2140,7 +2140,7 @@
         "@types/mocha": "^10.0.7",
         "@types/node": "^20.14.9",
         "@types/sinonjs__fake-timers": "^8.1.5",
-        "@types/vscode": "^1.90.0",
+        "@types/vscode": "1.89.0",
```

Comment:
Should this have been `"^1.89.0"`?

Reply:
OOF, that's a great catch. I think we need to revert this PR anyway to go back to our old engine requirement, making this moot, but it increases the urgency of us doing so. I'll take a look at doing this later today

Reply:
This was a correct change as the purpose of the pr was to make the extension available for 1.89. Specifying dependency version with a ^ would mean a newer types could be picked up from npm and type script would not error on a newer api usage. 

Having said that this safe to return to the previous state as the PR will be reverted for the next release.